index.html

```diff
@@ -1,9 +0,0 @@
-<!doctype html>
```

Comment:
I kept the `index.html` autogeneration because it integrates well in Webpack workflow, to make it simpler to understand and update if needed I push a template: https://github.com/rustwasm/rust-webpack-template/pull/7/files#diff-eacf331f0ffc35d4b482f1d15a887d3b

Reply:
Thanks!